### DISCUSSION TITLE
Make Foraging need Shears

### DISCUSSION THREAD

**Discussion by Sirfirelord69** [DiscordDungeons/Bugs#1052] [Ideas]
We finally got the item, now make it consistent and all the same

**Comment by FelixArgyle** [comment]
No.
level 1 will not be able to earn anything (only through adv which takes quite a while to accumulate 1.5k)

**Comment by Willard21** [comment]
You can pick lemons without shears. Chopping down trees, mining ore, and catching fish all need tools in real life. Shears were mainly added as a means to get items boosts and xp boosts, but it would be kinda weird for them to be required.

**Comment by Sven65** [comment]
Won't be implemented.
